config/settings.d/dns_nsupdate.yml.example

```diff
@@ -4,5 +4,7 @@
 #
 
 #:dns_key: /etc/rndc.key
-# use this setting if you are managing a dns server which is not localhost though this proxy
-#:dns_server: dns.domain.com
+
+# DNS server to talk to, usually localhost but can be arbitrary (remote) host.
+# This can be also set to blank value to send updates to the master server of the correct zone.
```

Comment:
Perhaps this could use a note that it uses nsupdate's built in support to automatically determine a master for a zone.